File: BOJ/1003_피보나치_함수/go/main.go
package main

import "fmt"

func main() {
	// 피보나치수 배열 만들기 0부터 40까지
	var fibonacciArr [41]int

	for i := 0; i <= 40 ; i++ {
		if i <= 1 {
			fibonacciArr[i] = i
		} else {
			fibonacciArr[i] = fibonacciArr[i - 1] + fibonacciArr[i - 2]
		}
	}
	var count int
	fmt.Scan(&count)

	// 크기가 count인 슬라이스 생성
	inputs := make([]int, count)
	for i := 0; i < count; i++ {
		fmt.Scan(&inputs[i])
	}

	for i := 0; i < count; i++ {
		if inputs[i] == 0 {
			fmt.Println("1 0")
		} else {
			fmt.Println(fibonacciArr[inputs[i] - 1], fibonacciArr[inputs[i]])
		}
	}
}

File: BOJ/1003_피보나치_함수/php/Main.php
<?php

for($i = 0; $i <= 40; $i++) {
    if($i <= 1)
        $fibonacciArr[$i] = $i;
    else {
        $fibonacciArr[$i] = $fibonacciArr[$i - 1] + $fibonacciArr[$i - 2];
    }
}
fscanf(STDIN,"%d",$count);

for($i = 0; $i < $count; $i++) {
    fscanf(STDIN,"%d",$inputs[$i]);
}
for($i = 0; $i < $count; $i++) {
    if($inputs[$i] == 0)
        fprintf(STDOUT,"1 0".PHP_EOL);
    else
        fprintf(STDOUT,"%d %d".PHP_EOL, $fibonacciArr[$inputs[$i] - 1], $fibonacciArr[$inputs[$i]]);
}
